Lib/idlelib/multicall.py

```diff
@@ -52,9 +52,9 @@
     _modifier_masks = (MC_CONTROL, MC_ALT, MC_SHIFT, MC_META)
 
 # a dictionary to map a modifier name into its number
-_modifier_names = dict([(name, number)
-                         for number in range(len(_modifiers))
-                         for name in _modifiers[number]])
+_modifier_names = {name: number
+                   for number in range(len(_modifiers))
+                   for name in _modifiers[number]}
```

Comment:
I prefer the change as easier to read and likely faster.  Ditto for the other dict(list) to dict comp change in this file.